Guides/Image_Formats.md

```diff
@@ -2,46 +2,44 @@
 
-### CloneCD Images (.CCD)
+### CloneCD Image (.CCD/.IMG)
 
-CCD is one step above CUE in terms of feature support, though it doesn't have nearly the same software support or customizability. It follows the same basic structure as CUE, with the .CCD file being the equivalent of the .CUE, and the .IMG file being the equivalent of the .BIN. On top of this, CCD also usually has a .SUB file, which is dedicated to storing the subchannel data of CDs. While the .CUE was essentially human-readable information about the dump's metadata and the disc's structure, the .CCD is a more accurate copy of the disc's structure and doesn't contain metadata in the same as CUE. 
+CCD images follow the same basic structure as CUE images, with the .CCD file as the equivalent of the .CUE, and the .IMG file as the equivalent of the .BIN. (In fact, these files should be identical for the same disc, as raw copies of the disc's data.) In addition, a CCD image usually contains a .SUB file, which is dedicated to storing the subchannel data of the CD. While the .CUE is essentially human-readable information about the dump's metadata and the disc's structure, the .CCD is a more accurate copy of the disc's structure and doesn't contain metadata in the same vein as CUE. In practice, a CCD image can be thought of a more rigidly defined CUE image that also supports subchannels; the format is one step above CUE in terms of feature support, but it can't be customized and doesn't have nearly the same software support.
 
-It's much more rigidly standard than CUE, having been created for the CloneCD program. Due to CloneCD's ability to create generally high quality dumps, with a support for the majority of optical based copy protections, this makes CCD one of the best formats to dump copy protected games to. So while CUE is moreso intended to be accessible for more use cases, CCD is much better for making good quality disc dumps intended to either be directly emulated in a virtual drive, or burned to a disc to play.
+As opposed to CUE, which is intended to be accessible for more use cases, CCD is much more rigidly defined as the output of the CloneCD program, which creates very high-quality dumps and supports the majority of optical-based copy protections. This makes CCD one of the best formats for copy-protected games, either for direct emulation in a virtual drive or for burning to a physical disc. However,  Also, for the same reasons as CUE, CCD and CloneCD only support CDs.
```

Comment:
The first sentence should say something closer to "rigidly defined due to being the output of...". Also, it's only CCD that doesn't support DVDs, CloneCD itself does.